tox.ini

```diff
@@ -1,6 +1,14 @@
 [tox]
-envlist = py27, py36
+envlist = py27, py38
```

Comment:
Hmm this is actually a big change right going from 3.6 to 3.8

Reply:
I can change it back but I will have to rely on github testing, was having issues getting pyenv to give tox python 3.6